Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/QuotasDbContext.cs

```diff
@@ -37,6 +38,7 @@ protected override void ConfigureConventions(ModelConfigurationBuilder configura
         configurationBuilder.Properties<TierId>().AreUnicode(false).AreFixedLength()
             .HaveConversion<TierIdEntityFrameworkValueConverter>();
 
+        configurationBuilder.Properties<MetricKey>().HaveConversion<MetricKeyEntityFrameworkValueConverter>();
```

Comment:
We should probably limit the length to... maybe 50? Should mit fixedLength = false and unicode = true